services/notify/null.go

```diff
@@ -197,6 +197,10 @@ func (*NullNotifier) SyncDeleteRef(ctx context.Context, doer *user_model.User, r
 func (*NullNotifier) RepoPendingTransfer(ctx context.Context, doer, newOwner *user_model.User, repo *repo_model.Repository) {
 }
 
+// NotifyNewUserSignUp notifies deletion of a package to notifiers
```

Comment:
copy-paste?
```suggestion
```

Alternatively, to conform with the rest
```suggestion
// NotifyNewUserSignUp places a place holder function
```